src/librustc_errors/emitter.rs

```diff
@@ -1379,7 +1379,6 @@ impl EmitterWriter {
                 for line in &annotated_file.lines {
                     for ann in &line.annotations {
                         span_left_margin = min(span_left_margin, ann.start_col);
-                        span_left_margin = min(span_left_margin, ann.end_col);
```

Comment:
What do these two changes do?

Reply:
Sorry, I was fine. I got confused start and end and didn't see the column. Now I turn it back.